### PR TITLE
Add some basic documentation

### PR DESCRIPTION
**Pull request by mkolesnik**
Added some very basic docs how to use shipyard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,58 @@ The Shipyard project provides tooling for creating K8s clusters with [kind](K8s
 - [go 1.12] with [$GOPATH configured]
 - [docker]
 
+## Usage
+To use shipyard for your project, it's easiest to use dapper and make.
```
Comment (dfarrell07) [inline README.md:14]:
```suggestion
To use Shipyard for your project, it's easiest to use Dapper and Make.
```

File: README.md [hunk 2]
```diff
@@ -10,6 +10,58 @@ The Shipyard project provides tooling for creating K8s clusters with [kind](K8s
...
+To use dapper you'll need a specific Dockerfile that dapper consumes to create a consistent environment based upon shipyard's base image.
```
Comment (dfarrell07) [inline README.md:15]:
```suggestion
To use Dapper, you'll need a specific Dockerfile that Dapper consumes to create a consistent environment based upon Shipyard's base image.
```

File: README.md [hunk 3]
```diff
@@ -10,6 +10,58 @@ The Shipyard project provides tooling for creating K8s clusters with [kind](K8s
...
+To use make you'll need some commands to enable dapper and also include the targets which ship in the base image.
```
Comment (dfarrell07) [inline README.md:16]:
```suggestion
To use Make, you'll need some commands to enable Dapper and also include the targets which ship in the base image.
```

File: README.md [hunk 4]
```diff
@@ -10,6 +10,58 @@ The Shipyard project provides tooling for creating K8s clusters with [kind](K8s
...
+
+### Dockerfile.dapper
+
+The dockerfile should build upon `quay.io/submariner/shipyard-dapper-base`.
```
Comment (dfarrell07) [inline README.md:20]:
```suggestion
The Dockerfile should build upon `quay.io/submariner/shipyard-dapper-base`.
```

File: README.md [hunk 5]
```diff
@@ -10,6 +10,58 @@ The Shipyard project provides tooling for creating K8s clusters with [kind](K8s
...
+
+ENTRYPOINT ["./scripts/entry"]
+CMD ["ci"]
+```
+
+You can also refer to the project's own [Dockerfile.dapper](Dockerfile.dapper) as an example.
+
+### Makefile
+
+The makefile should include targets to run everything in dapper, they're defined in [Makefile.dapper](Makefile.dapper) and can be copied as is and included. To use shipyard's target simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
```
Comment (dfarrell07) [inline README.md:40]:
```suggestion
The Makefile should include targets to run everything in Dapper. They're defined in [Makefile.dapper](Makefile.dapper) and can be copied as-is and included. To use Shipyard's target, simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
```

File: README.md (hunk 1 above)
Comment (mangelajo) [reply README.md:14]: dapper and make are normally called in low-case from the cmdline, for those may be it's better to use lowcase?

File: README.md (hunk 4 above)
Comment (mangelajo) [reply README.md:20]: Dockerfile uses upcase :+1:

File: README.md (hunk 1 above)
Comment (dfarrell07) [reply README.md:14]: I think if we want to use literal commands, we should use `make` or `dapper`. If we are talking about the tools themselves, I think they are proper nouns and should be Capitalized.

Comment (dfarrell07) [reply README.md:14]: I agree that there are a lot of cases of them lowercase though, so definitely a non-blocker. Wikipedia uses it as I described I think. https://en.wikipedia.org/wiki/Make_(software)

**Comment by mkolesnik** [comment]
Thanks Daniel, I've committed your suggestions

**Pull request merged**